policyengine_api/data/data.py

```diff
@@ -260,4 +276,4 @@ def set_in_table(
             self.query(updater, tuple(update.values()) + tuple(match.values()))
 
 
-database = PolicyEngineDatabase(local=False, initialize=True)
+database = PolicyEngineDatabase(local=True, initialize=True)
```

Comment:
just a reminder to not merge this as-is (since this isn't a draft pr)

Reply:
Oh good catch, thanks